docs/cvl/invariants.md

```diff
@@ -75,6 +79,19 @@ and by {ref}`--parametric_contracts`. `View` and `pure` methods are excluded fro
 invariant checking as by definition they cannot change the state 
 of their contract. 
 
+A strong invariant performs the same checks as a weak invariant - i.e. it will be checked for the constructor
+and for any other method it will be assumed before executing the method (pre-state) and asserted afterward execution of the method (post-state). 
```

Comment:
```suggestion
and for any other method, it will be assumed before executing the method (pre-state) and asserted afterward execution of the method (post-state). 
```